Source/WebKit/UIProcess/API/glib/WebKitURISchemeRequest.cpp

```diff
@@ -204,6 +205,26 @@ SoupMessageHeaders* webkit_uri_scheme_request_get_http_headers(WebKitURISchemeRe
     return request->priv->headers.get();
 }
 
+/**
+ * webkit_uri_scheme_request_get_http_body:
+ * @request: a #WebKitURISchemeRequest
+ *
+ * Get the request body.
+ *
+ * Returns: (transfer none): (nullable): the body of the @request.
```

Comment:
I don't think transfer none is useful here, the returned stream is not seekable, so after being read, if you call this again, you will get a stream that can't be read again.

Reply:
I'll change to transfer full and just return the stream to user then.